### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by xiaolou86**

### CODE REVIEW

**Review by tersec — APPROVED** [review]

File: tests/test_beacon_time.nim
Comment (tersec) [inline tests/test_beacon_time.nim:1]: `Copyright (c) 2022 Status Research & Development GmbH` needs to be `Copyright (c) 2022-2024 Status Research & Development GmbH` to satisfy the CI.

File: scripts/signers/custom.sh
Comment (tersec) [inline scripts/signers/custom.sh:1]: `# Copyright (c) 2023 Status Research & Development GmbH.` needs to be `# Copyright (c) 2023-2024 Status Research & Development GmbH.` to satisfy the CI.

**Comment by tersec** [comment]
```
Run excluded_files="config.yaml"
The following files do not have an up-to-date copyright year:
- scripts/signers/custom.sh
- tests/test_beacon_time.nim
Error: Process completed with exit code 2.
```

**Comment by xiaolou86** [comment]
> ```
> Run excluded_files="config.yaml"
> The following files do not have an up-to-date copyright year:
> - scripts/signers/custom.sh
> - tests/test_beacon_time.nim
> Error: Process completed with exit code 2.
> ```

@tersec thanks for your review.

**Pull request merged**